tests/e2e/pageObjects/memory-efficiency-page.ts

```diff
@@ -58,27 +54,48 @@ export class MemoryEfficiencyPage extends InstancePage {
     // CONTAINERS
-    luaScriptAccordion = Selector('[data-testid=luaScript-accordion]');
-    luaScriptTextContainer = Selector('#luaScript');
-    useSmallKeysAccordion = Selector('[data-testid=useSmallerKeys-accordion]');
-    bigHashesAccordion = Selector('[data-testid=bigHashes-accordion]');
-    combineStringsAccordion = Selector('[data-testid=combineSmallStringsToHashes-accordion]');
-    increaseSetAccordion = Selector('[data-testid=increaseSetMaxIntsetEntries-accordion]');
-    avoidLogicalDbAccordion = Selector('[data-testid=avoidLogicalDatabases-accordion]');
-    convertHashToZipAccordion = Selector('[data-testid=convertHashtableToZiplist-accordion]');
-    compressHashAccordion = Selector('[data-testid=compressHashFieldNames-accordion]');
     veryUsefulVoteBtn = Selector('[data-testid=very-useful-vote-btn]').nth(0);
     usefulVoteBtn = Selector('[data-testid=useful-vote-btn]').nth(0);
     notUsefulVoteBtn = Selector('[data-testid=not-useful-vote-btn]').nth(0);
     recommendationsFeedbackBtn = Selector('[data-testid=recommendation-feedback-btn]');
-    toTutorialsBtn = Selector('[data-testid=RTS-to-tutorial-btn]');
-    rtsAccordeon = Selector('[data-testid=RTS-accordion]');
 
     /**
-     * Find recommendation selector by name
-     * @param name A recommendation name
+     * Get recommendation selector by name
+     * @param recommendationName Name of the recommendation
      */
-    async getRecommendationByName(name: string): Promise<Selector> {
-        return Selector('div').withExactText(name).parent('[data-testid=RTS-accordion]').parent();
+    getRecommendationByName(recommendationName: RecommendationIds): Selector {
+        return Selector(`[data-testid=${recommendationName}-recommendation]`);
+    }
+
+    /**
+     * Get recommendation label by recommendation name
+     * @param recommendationName Name of the recommendation
+     * @param label Label of changes
+     */
+    getRecommendationLabelByName(recommendationName: RecommendationIds, label: string): Selector {
+        return this.getRecommendationByName(recommendationName).find(`[data-testid=${label}_changes]`);
+    }
+
+    /**
+     * Check if the recommendation exist
+     * @param recommendationName Name of the recommendation
+     */
+    async isRecommendationExists(recommendationName: RecommendationIds): Promise<boolean> {
+        return this.getRecommendationByName(recommendationName).exists;
+    }
+
+    /**
+     * Get recommendation expand/collapse button by recommendation name
+     * @param recommendationName Name of the recommendation
+     */
+    getRecommendationButtonByName(recommendationName: RecommendationIds): Selector {
+        return Selector(`[data-test-subj=${recommendationName}-button]`);
+    }
+    /**
+     * Get recommendation Tutorial button by recommendation name
+     * @param recommendationName Name of the recommendation
+     */
+    getToTutorialBtnByRecomName(recommendationName: RecommendationIds): Selector {
```

Comment:
probably getTutorialButtonRecommendationName?

Reply:
I think that getToTutorialBtnByRecomName is good, because we find Tutorial button by recommendation name here